spec/models/events/comment_replied_to_spec.rb

```diff
@@ -9,16 +9,23 @@
 
   describe "::publish!" do
     let(:event) { double(:event, notify_users!: true) }
+    let(:mailer) { ThreadMailer.new }
     before { Event.stub(:create!).and_return(event); comment }
+    before { ThreadMailer.stub_chain(:delay, :comment_replied_to) }
 
     it 'creates an event' do
       Event.should_receive(:create!).with(kind: 'comment_replied_to',
                                           eventable: comment,
-                                          user: comment.author,
```

Comment:
Why this change in functionality? I think it makes more sense on events for the user to be the 'actor', rather than the recipient of the notification.